Description: Downgrade ACPI shutdown error to debug message to fix tests
Author: Colin Watson <cjwatson@ubuntu.com>
Forwarded: yes
Last-Update: 2010-11-23

Index: b/grub-core/commands/acpihalt.c
===================================================================
--- a/grub-core/commands/acpihalt.c
+++ b/grub-core/commands/acpihalt.c
@@ -324,6 +324,6 @@
 	}
     }
 
-  grub_printf ("ACPI shutdown failed\n");
+  grub_dprintf ("acpi", "ACPI shutdown failed\n");
 }
 #endif
